pkg/services/sqlstore/permissions/dashboard.go

```diff
@@ -54,7 +52,7 @@ func (d DashboardPermissionFilter) Where() (string, []interface{}) {
 							-- include default permissions -->
 							da.org_id = -1 AND (
 							  (folder.id IS NOT NULL AND folder.has_acl = ` + falseStr + `) OR
-							  (folder.id IS NULL AND d.has_acl = ` + falseStr + `)
+							  (folder.id IS NULL AND d.has_acl = ` + falseStr + `) 
```

Comment:
looks like a trailing space was added by accident here

Reply:
not sure why this comment was confusing, this change just adds an unneeded space at the end of the line.  this kind of change is just noise in this PR andthe next PR that touches this file from any user with an editor that trims trailing whitespace (which will remove it again).

Reply:
I put the icon trying to say that I don't know how I added it (I passed the lint and I didn't have any trailing space in my code), not because it's confusing. I understood the comment and sorry for confusing you with that 😞.